src/wp-admin/includes/class-wp-posts-list-table.php

```diff
@@ -1238,7 +1251,7 @@ protected function handle_row_actions( $post, $column_name, $primary ) {
 				__( 'Edit' )
 			);
 			$actions['inline hide-if-no-js'] = sprintf(
-				'<button type="button" class="button-link editinline" aria-label="%s" aria-expanded="false">%s</button>',
+				'<a href="#" class="editinline" aria-label="%s">%s</a>',
```

Comment:
What's the reason for including this change?

Reply:
Hi @nylen, I didn't touch that line. I forked from the last available version just before doing the PR. Is it possible that the CP [release that happened yesterday](https://forums.classicpress.net/t/introducing-classicpress-1-3-0/3516/2) is related? I have seen that it includes a lot of accessibility improvements, so this exact lines you show should be omited in the merge.